anaUltraScurve.py

```diff
@@ -208,18 +223,73 @@ def overlay_fit(VFAT, CHAN):
     vthr_list[event.vfatN][event.vfatCH] = event.vthr
     trim_list[event.vfatN][event.vfatCH] = event.trimDAC
     trimrange_list[event.vfatN][event.vfatCH] = event.trimRange
+    if options.SaveFile:
+        fitter.feed(event)
+        pass
     pass
 
 if options.SaveFile:
+    scanFits = fitter.fit()
+    pass
+
+# Determine hot channels
+import numpy as np
+if options.SaveFile:
+    print 'Determining hot channels'
     masks = []
+    maskReasons = []
```

Comment:
The tuple thing isn't going to play well with the bitmasks. I added an enum-like class in `anaInfo.py` to address the readability/maintainability concern. There is also a `humanReadable(reason)` function there which returns e.g. `'HotChannel,FitFailed'`.